packages/react/src/Button/types.ts

```diff
@@ -76,6 +76,8 @@ export type ButtonProps = {
 
 export type IconButtonProps = ButtonA11yProps & {
   icon: React.ElementType
+  disableTooltip?: boolean
```

Comment:
Name ideas: I think the name should communicate that this is a temporary workaround or a hack and not a permanent feature

ideas: `temp_disableTooltip` or `hack_disableTooltip`. I like the putting the word hack in there for extra awkardness

Reply:
happy with `UNSAFE_disableTooltip`!

Reply:
Okay, `UNSAFE_disableTooltip` it is! So out of convention but very discouraging as the way we want it.

Reply:
Oh, underscore hits the eslint camel case error rule though 😢 in primer/react we have it as error and [most of the packages at dotcom](https://github.com/search?q=repo%3Agithub%2Fgithub+camelcase+language%3AJavaScript&type=code&l=JavaScript) have it as error too. Should we use a name without underscore like `UnsafeDisableTooltip` or any other suggestions to overcome this violation?